frontend/src/pages/Users/Users.tsx

```diff
@@ -288,10 +288,12 @@ export const Users: React.FC = () => {
               );
             } catch (e) {
               // Just continue when an error occurs
-              console.error(e);
+              console.error('Error loading entry: ' + e);
             }
           }
           setUsers(users.concat(...createdUsers));
+          //without location reload site will not refresh to update data
```

Comment:
@ValeriaReveles , we shouldn't need to run `window.location.reload()` to get data to refresh. The `setUsers(users.concat(...createdUsers));` should automatically refresh the data, because it will update the React state and the table will change.

Reply:
Happy to chat more about this to debug what might be going on!

Reply:
Currently, `setUsers(users.concat(...createdUsers));` only returns "updated" users and not newly "created" users without having that `window.location.reload()` to refresh the rest of the data. This is due to how it is programmed on the backend, therefore for functionality to work as desired- either the frontend or backend needs to be changed accordingly. I recommend leaving that `window.location.reload()` since it resolves the issue of page and data not refreshing/rendering, then creating a separate issue to address the root cause and implement a more optimal solution. 

Please advise what course of action/solutions you want me to look into. Thank you in advance!

Reply:
> Currently, `setUsers(users.concat(...createdUsers));` only returns "updated" users

Yeah, but since it includes updated users, wouldn't it also include the newly created users?

Let's chat more about this to see what might be going on though! Maybe I'm missing something

Reply:
Sounds good! Please let me know a time that works best for you.